fdbclient/S3Client_cli.actor.cpp

```diff
@@ -32,6 +35,7 @@
 #include <io.h>
 #endif
 
+#include <boost/algorithm/hex.hpp>
 #include "fdbclient/BuildFlags.h"
 #include "fdbclient/BackupContainerFileSystem.h"
 #import "fdbclient/BackupTLSConfig.h"
```

Comment:
The first c++ module import? I like the idea of modules, which can make compiling faster.

You probably can experiment with importing other modules, e.g., std.

Reply:
I think this was more an accident than intentional. Leaving it in though. Smile.